main.py

```diff
@@ -575,6 +575,36 @@ def delete_tag_range(self, request: Request) -> JSONResponse:
             raise HTTPException(400, detail=detail)
         return JSONResponse("Operation Successful", status_code=200)
 
+    @rest('v3/interfaces/tag_ranges', methods=['GET'])
+    @validate_openapi(spec)
+    def get_all_tag_ranges(self, _):
```

Comment:
Trivial detail but if you could add the type hints in the arg and return type that'd be great like we have on the other endpoints.